src/plugins/chart_expressions/expression_xy/public/helpers/data_layers.tsx

```diff
@@ -297,18 +296,10 @@ export const getSeriesName: GetSeriesNameFn = (
   return splitValues.length > 0 ? splitValues.join(' - ') : yAccessorTitle;
 };
 
-const getPointConfig: GetPointConfigFn = ({
-  xAccessor,
-  markSizeAccessor,
-  emphasizeFitting,
-  showPoints,
-  pointsRadius,
-}) => {
+const getPointConfig: GetPointConfigFn = ({ markSizeAccessor, showPoints, pointsRadius }) => {
   return {
-    visible: (showPoints !== undefined ? showPoints : !xAccessor || markSizeAccessor !== undefined)
-      ? 'always'
-      : 'never',
-    radius: pointsRadius !== undefined ? pointsRadius : xAccessor && !emphasizeFitting ? 5 : 0,
+    visible: showPoints || markSizeAccessor ? 'always' : 'never',
```

Comment:
simplifying this, points should be visible if showPoints is true or markSizeAccessor exist. we don't need a check on the `xAccessor`